.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@
 
 ## Does this affect other platforms - Amp, Apps, etc?
 
+## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?
```

Comment:
💯 would be good to guide people to examples

Maybe a comment with a link to the tone/advertisement-features front?

```
## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

<!-- content can be found here: https://www.theguardian.com/tone/advertisement-features -->
```